testplans/lotus-soup/manifest.toml

```diff
@@ -9,8 +9,8 @@ enabled = true
 
 [builders."docker:go"]
 enabled = true
-build_base_image = "iptestground/oni-buildbase:v13-lotus"
-runtime_image = "iptestground/oni-runtime:v8-debug"
+build_base_image = "iptestground/oni-buildbase:v14-lotus"
+runtime_image = "iptestground/oni-runtime:v9-debug"
```

Comment:
I've upgraded docker images with latest `lotus` and `lotus-miner` binaries (aliased to `lll` and `lm` within the images), so that we can debug deadlocked devnets, when a deal ends up in the `StorageDealAwaitingPreCommit` state.